packages/shared/src/graphql/posts.ts

```diff
@@ -28,15 +29,21 @@ export enum PostType {
   Share = 'share',
   Welcome = 'welcome',
   Freeform = 'freeform',
+  Collection = 'collection',
 }
 
-export const internalReadTypes: PostType[] = [PostType.Welcome];
+export const internalReadTypes: PostType[] = [
+  PostType.Welcome,
+  PostType.Freeform,
```

Comment:
Just making sure this addition of freeform here is intentional, since not related to collections

Reply:
Yes, actually it is intended since freeform does not have external URL so can be read only on our platform. Good eye though 👌

Reply:
Basically fixes freeform posts having this read icon on the top 
<img width="340" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/0a3b05d9-c602-4bc7-9010-deaef5068027">